docs/source/resources/index.rst

```diff
@@ -0,0 +1,27 @@
+.. _resources:
+
+Resources
+=========
+
+Example configuration YAMLs
+---------------------------
+
+`Preprocessing <https://storage.cloud.google.com/bw-cellmincer-dev/configs/preprocess.yaml>`_
+
+`Training <https://storage.cloud.google.com/bw-cellmincer-dev/configs/train.yaml>`_
+
+Trained models
+--------------
+
+`Trained on 5 Optosynth datasets <https://storage.cloud.google.com/bw-cellmincer-dev/models/optosynth.ckpt>`_
+
+`Trained on 10 Optopatch datasets <https://storage.cloud.google.com/bw-cellmincer-dev/models/optopatch_10.ckpt>`_ [#farhi]_
+
+`Trained on 26 (high-speed dye) Optopatch datasets <https://storage.cloud.google.com/bw-cellmincer-dev/models/optosynth.ckpt>`_ [#miller]_
```

Comment:
Rename like above.

Is the data available for download? If it is, then provide a description about how the data could be obtained.